web/src/frontend/scripts/mock_data/soknadsosialhjelp_nb_NO.properties

```diff
@@ -1,5 +1,5 @@
-arbeid.dinsituasjon.annensituasjon.true.feilmelding=Du må beskrive situasjonen din kort
-arbeid.dinsituasjon.annensituasjon.true.sporsmal=Beskriv kort
+arbeid.dinsituasjon.annensituasjon.true.beskrivelse.feilmelding=Du må beskrive situasjonen din kort
+arbeid.dinsituasjon.annensituasjon.true.beskrivelse.sporsmal=Beskriv kort
```

Comment:
Vi får lage oss et lite script som lager en liten tekstfil for hver nøkkel i denne filen.